### PR TITLE
Update the doc as part of the schema changes done in PR#230

### PR DESCRIPTION
**Pull request by ramya-bangera**
1. **What this PR does / why we need it**:
  Schema changes are added as part of [PR](https://github.com/vmware/terraform-provider-tanzu-mission-control/pull/230), this PR contains the corresponding doc changes.

2. **Which issue(s) this PR fixes**

        (optional, in `fixes #<issue number>` format, will close the issue(s) when PR gets merged):

        Fixes # Issue reported over slack

3. **Additional information**

4. **Special notes for your reviewer**

### CODE REVIEW

File: docs/data-sources/cluster_node_pool.md [hunk 1]
```diff
@@ -91,6 +91,22 @@ Required:
 - `class` (String) Nodepool instance type
 - `storage_class` (String) Storage Class to be used for storage of the disks which store the root filesystem of the nodes
 
+Optional:
+
+- `failure_domain` (String) Configure the failure domain of node pool. The potential values could be found using cluster:options api. This parameter will be ignored by the backend if the TKG service vsphere cluster doesn't support.
```
Comment (vmw-vjn) [inline docs/data-sources/cluster_node_pool.md:96]: I think we can rephrase the portion as it feels ambiguous. 
`This parameter will be ignored by the backend if the TKG service vsphere cluster doesn't support.`

Suggest instead:
> Failure domains, also known as vSphere Fault Domains, are defined and managed by the vSphere administrator when creating vSphere Zones. And the parameter will be ignored if failure domains doesn't exist or enabled in vSphere.

File: docs/resources/cluster_node_pool.md [hunk 2]
```diff
@@ -110,6 +114,22 @@ Required:
 - `class` (String) Nodepool instance type
 - `storage_class` (String) Storage Class to be used for storage of the disks which store the root filesystem of the nodes
 
+Optional:
+
+- `failure_domain` (String) Configure the failure domain of node pool. The potential values could be found using cluster:options api. This parameter will be ignored by the backend if the TKG service vsphere cluster doesn't support.
```
Comment (vmw-vjn) [inline docs/resources/cluster_node_pool.md:119]: I think we can rephrase the portion as it feels ambiguous. 
`This parameter will be ignored by the backend if the TKG service vsphere cluster doesn't support.`

Suggest instead:
> Failure domains, also known as vSphere Fault Domains, are defined and managed by the vSphere administrator when creating vSphere Zones. And the parameter will be ignored if failure domains doesn't exist or enabled in vSphere.

File: docs/data-sources/cluster_node_pool.md (hunk 1 above)
Comment (ramya-bangera) [reply docs/data-sources/cluster_node_pool.md:96]: This description is picked up from the [api repo ](https://gitlab.eng.vmware.com/olympus/api/-/blob/main/proto/manage/v1alpha1/cluster/nodepool/nodepool.proto#L169), have copied the same just to keep consistent

**Comment by ramya-bangera** [comment]
This change is also included as part of PR#232[https://github.com/vmware/terraform-provider-tanzu-mission-control/pull/232](url), hence closing this PR.

@vmw-vjn - lets move the open conversation to new PR[https://github.com/vmware/terraform-provider-tanzu-mission-control/pull/232](url)

**Pull request merged**